spec/donation_system/data_structs_for_tests.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module DonationSystem
+  RawRequestData = Struct.new(:amount, :currency, :giftaid, :token,
+                              :name, :email,
+                              :address, :city, :state, :zip, :country)
+  RawPaymentData = Struct.new(:status, :amount)
+  SupporterSObjectFake = Struct.new(:AccountId)
+
+  VALID_REQUEST_DATA = RawRequestData.new(
+    '12.345', 'gbp', true, 'tok_visa', 'Name Lastname', 'user@example.com',
```

Comment:
rename to `token_visa` and `Firstname Lastname`?

Reply:
I can not rename the token, it's a test value from Stripe:

![test](https://user-images.githubusercontent.com/2157089/32961522-c5e752a8-cbc0-11e7-9f64-18f63c5fe6e0.png)

Will rename the `Name`.